src/som/vmobjects/SArray.java

```diff
@@ -113,6 +115,79 @@ public Object getStoragePlain() {
       return this.getObjectStorage(ValueProfile.createClassProfile());
     }
   }
+  
+  public void convertAllToObject(){
```

Comment:
[This?](https://github.com/charig/TruffleMATE/commit/e5b2d674a4cbd4d3177516cae29c602ebef47701)